Makefile

```diff
@@ -12,18 +12,27 @@ IDENTIFIER=$(GOOS)-$(GOARCH)
 help:  ## Show available options with this Makefile
 	@grep -F -h "##" $(MAKEFILE_LIST) | grep -v grep | awk 'BEGIN { FS = ":.*?##" }; { printf "%-15s  %s\n", $$1,$$2 }'
 
+web:
+	@cd dashboard/web && yarn build && cp -a ./dist ../../web
+
+yarn:
+	@rm -rf web
+	@cd dashboard/web && yarn build && cp -a ./dist ../../web
```

Comment:
If this is intended to be the same as `web`, we can use `make web` for this line

Reply:
What's the difference between `make web` and `make yarn` by design?

Reply:
make all: first `@rm -rf web` then build `web` once
make yarn: `@rm -rf web` only build `web`, not build `nknc` `nknd`
if `@rm -rf web` then `make all` will build `web` before every platform.
I don't know how to code `make web`. Could you help me?

Reply:
We need two make target:
* rebuild web dir with no excuse
* rebuild web dir only case in resource files was updated